longtailstorelib/remotestore.go

```diff
@@ -55,9 +55,6 @@ type getExistingContentMessage struct {
 	asyncCompleteAPI     longtaillib.Longtail_AsyncGetExistingContentAPI
 }
 
-type stopMessage struct {
```

Comment:
Removed the stop message and replaced it with checking the open/close flag when fetching memory.

For the block worker it checks the putBlock channel since that is the ciritical path. Any other channel does not affect exit status.

For the index worker it checks the blockIndex channel - the blockIndex channel is what matters to that worker in regards to exit status.